tests/runmode-test/test.yaml

```diff
@@ -0,0 +1,13 @@
+requires:
+  min-version: 7.0.3 
+  pcap: false
+
+exit-code: 1
+
+args: 
+  - -c suricata.yaml
+
+checks:
+
+  - file:
+     args: "No capture mode specified as a command-line argument, specify one from the --list-runmodes" stderr
```

Comment:
I don't think this works as intended. Please take a better look at the example from https://github.com/OISF/suricata-verify/blob/master/tests/detect-strip_whitespace-01/test.yaml#L9-L12 
to understand how to check for an error message in the stderr file and count its occurrences ;)

Reply:
> I don't think this works as intended. Please take a better look at the example from https://github.com/OISF/suricata-verify/blob/master/tests/detect-strip_whitespace-01/test.yaml#L9-L12 to understand how to check for an error message in the stderr file and count its occurrences ;)

Thank you for your feedback. I've carefully reviewed the suggestion. And I have verified that the error message in the `stderr` file matches the specified one exactly, and there are no extra occurrences.  On using `grep` and `wc` in the checks like the example you gave. In my case, the error message occurs exactly once in the `stderr` file, so I did not explicitly specify `expect: 1`.
 If it's beneficial for clarity, I can include it, but I wanted to confirm if my understanding is correct and if it's necessary in this specific scenario.

Reply:
I understand your reasoning, but have you run this test locally to see if it works as expected?

What I meant to say with my other comment is that the check itself doesn't work as is. Since we are missing the `grep` there, there isn't something for SV to compare against and indicate whether the test fails or succeeds, and therefore it doesn't interpret this as a test.  Does this help?

Reply:
That is very possible, one thing I still want to improve with SV is to get better error messages when a test fails to run, instead of just showing nothing, as it happens now and has happened to you. Glad to know that things are making sense! :)

Reply:
> That is very possible, one thing I still want to improve with SV is to get better error messages when a test fails to run, instead of just showing nothing, as it happens now and has happened to you. Glad to know that things are making sense! :)

Thank you. So do I go ahead and incorporate this feedback or I wait out for the PR changes

Reply:
The format of the test itself needs to be updated, regardless of future decisions on what should the output of that command look like. So I think you are safe to incorporate these changes, but can hold on submitting a new PR for them, for now.